code/game/gamemodes/devil/devilinfo.dm

```diff
@@ -435,5 +435,6 @@ var/global/list/lawlorify = list (
 	soulCounter = SOULVALUE
 	var/obj/screen/devil/soul_counter/K = new()
 	K.update()
+	var/mob/living/carbon/D = new()
```

Comment:
Alrighty, thanks dude, it doesn't quite work, but I'll figure it out. You helped me quite a long way